File: ProjectEulerPython/problems/problem_017.py
from problems.problem import Problem
import str_helpers

class Problem017(Problem):

  def calculate_answer(self) -> int:
    answer = 0
    n = 1000

    for i in range(1, n + 1):
      answer += len(str_helpers.int_to_words(i).replace(" ", ""))

    return answer

File: ProjectEuler/Problems/Problem017.cs
using Microsoft.Extensions.Logging;

namespace ProjectEuler.Problems
{
    public class Problem017 : Problem<int>
    {
        public Problem017(ILogger<Problem017> logger) : base(logger) { }

        protected override int CalculateAnswer()
        {
            var answer = 0;

            for (var i = 1; i <= 1000; ++i)
            {
                answer += i.ToWords().Replace(" ", "").Length;
            }

            return answer;
        }
    }
}
